README.mediawiki

```diff
@@ -301,6 +301,14 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Standard
 | Draft
 |-
+| [[bip-0053.mediawiki|53]]
+| Consensus (soft fork)
+| Disallow 64-byte transactions
+| Chris Stewart
+| Standard
+| Draft
+<!-- 50 series reserved for a group of post-mortems -->
```

Comment:
this line seems outdated and if anywhere, it should be after BIP 50 or after BIP 54.

```suggestion
```